shared/src/main/scala/com/cibo/evilplot/plot/Histogram.scala

```diff
@@ -59,7 +59,8 @@ object Histogram {
     barRenderer: BarRenderer,
     binCount: Int,
     spacing: Double,
-    boundBuffer: Double
+    boundBuffer: Double,
+    normalized: Boolean = false
```

Comment:
Defaulting `normalized = false` means that it always gets set to false at render time, even when if it was true in the apply. i.e `Histogram(Seq(1, 2, 3, 3), 3, normalized = true)` is always rendered incorrectly, because the `createBins` method below returns the wrong point y values, which then messes up the `yscale`. To fix this, make sure `normalized` gets passed into the HistogramRenderer in the apply method.